scss/initial/_sample-previews.scss

```diff
@@ -1,10 +1,10 @@
-// Herman Custom Properties for <iframe> Previews
-// ==============================================
+// Sample Previews
+// ===============
 
-@use 'config';
 @use '~accoutrement/sass/tools';
 @use 'samples/colors';
 
+// Add color sample CSS custom properties
```

Comment:
@dvdherron @mirisuzanne Instead of using a custom file and the new `customPreviewCSS` option, I moved this to our main CSS file. I guess we're not setting `font-family` anywhere in Herman app styles? But if we would, that would now be included in the iframes.

Reply:
> I guess we're not setting `font-family` anywhere in Herman app styles? But if we would, that would now be included in the iframes.

We are but by using the Accoutrement mixin (e.g. `@include tools.font-family('sans')`). Are these not being caught by the changes you made?

Reply:
Oh, I guess it is, I was wrong. This is what it's pulling in:

```
:root {
  font-family:Source Sans Pro,Helvetica Neue,Helvetica,Arial,sans-serif;
  font-size:18px;
  font-size:calc(1em + .125vw);
  --herman-brand-blue:#0d7fa5;
  --herman-brand-pink:#e2127a;
}
```